workflow/rules/report.smk

```diff
@@ -1,4 +1,5 @@
 rule vcf_report:
+    threads: 8
```

Comment:
Move threads directly above shell. Not enforced by snakefmt yet, but this is more consistent.